src/features/lightspeed/errors.ts

```diff
@@ -184,11 +184,18 @@ ERRORS.addError(
     "Could not find an API Key for IBM watsonx Code Assistant. Please contact your administrator.",
   ),
 );
+ERRORS.addError(
+  403,
+  new Error(
+    "error__wca_organization_not_linked",
+    "Your user Id is not linked to an organization Id. Please contact your administrator.",
```

Comment:
This situation should never happen by design, we should instead ask the user to report a bug.

Reply:
@goneri I think end users need to contant their adminitrator to open a ticket.  Does the following message sound reasonable?
```
"Your user Id is not linked to an organization Id. Please contact your administrator to open a ticket with Red Hat support."
```

Reply:
Agree with @goneri 's comment. If this ever happened with our saas Lightspeed service, it's a bug somewhere, and if it happens with on-prem, it's a misconfiguration of the service. How about something like "Ansible Lightspeed is not properly configured to determine your model id. Please contact your administrator." to cover either scenario?

Reply:
Thinking on it more, I think wisdom-service should be updated to allow a custom model override even when a default model is not configured with ANSIBLE_AI_MODEL_MESH_MODEL_NAME. Then this message could be something like "Ansible Lightspeed does not have a model configured. Contact your Ansible administrator to configure a model, or provide a model id override in your VSCode Ansible extension settings." This is a very on-prem message, with the expectation we'd never hit this particular error with saas Lightspeed. wdyt?

Reply:
As I wrote in the PR on the service part, if it is possible to have a user without a link to an organization, the exception thrown will be changed. Let me work on the service part first and change the extension later.

Reply:
The message was updated in the latest commit, which corresponds to the changes on the service side to add WcaNoDefaultModelId exception.